usage/dist/src/main/dist/pid_java

```diff
@@ -0,0 +1 @@
+83209
```

Comment:
Great, will remove then update.

Thank you!

On 1 October 2015 at 06:16, Alex Heneveld <notifications@github.com> wrote:

> In usage/dist/src/main/dist/pid_java
> <https://github.com/apache/incubator-brooklyn/pull/928#discussion_r40911191>
> :
>
> > @@ -0,0 +1 @@
> > +83209
>
> this file should be removed. i think it's the one causing the RAT error.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-brooklyn/pull/928/files#r40911191>.
>

-- 
Cloudsoft Corporation Limited, Registered in Scotland No: SC349230. 
 Registered Office: 13 Dryden Place, Edinburgh, EH9 1RP
 
This e-mail message is confidential and for use by the addressee only. If 
the message is received by anyone other than the addressee, please return 
the message to the sender by replying to it and then delete the message 
from your computer. Internet e-mails are not necessarily secure. Cloudsoft 
Corporation Limited does not accept responsibility for changes made to this 
message after it was sent.

Whilst all reasonable care has been taken to avoid the transmission of 
viruses, it is the responsibility of the recipient to ensure that the 
onward transmission, opening or use of this message and any attachments 
will not adversely affect its systems or data. No responsibility is 
accepted by Cloudsoft Corporation Limited in this regard and the recipient 
should carry out such virus and other checks as it considers appropriate.